File: Debug/Barcode/Examples/VS2010/RDLC_Windows_VB/Form1.vb

Imports System.Security
Imports System.Security.Permissions

Public Class Form1

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load

        Dim permissions As New PermissionSet(PermissionState.Unrestricted)
        Me.ReportViewer1.LocalReport.SetBasePermissionsForSandboxAppDomain(permissions)

        Me.CustomerBindingSource.DataSource = New Customer()
        Me.LineItemBindingSource.DataSource = New List(Of LineItem)()
    End Sub

    Private Sub viewReportButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles viewReportButton.Click
        Me.ReportViewer1.RefreshReport()
    End Sub
End Class

File: Debug/Barcode/Examples/VS2013/RDLC_Windows_CS/Form1.cs
using System;
using System.Collections.Generic;
using System.Security;
using System.Security.Permissions;
using System.Windows.Forms;

namespace RDLC_Windows_CS
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            PermissionSet permissions = new PermissionSet(PermissionState.Unrestricted);
            this.ReportViewer1.LocalReport.SetBasePermissionsForSandboxAppDomain(permissions);
            this.CustomerBindingSource.DataSource = new Customer();
            this.LineItemBindingSource.DataSource = new List<LineItem>();
        }

        private void viewReportButton_Click(object sender, EventArgs e)
        {
            this.ReportViewer1.RefreshReport();
        }
    };
}
